examples/example6.c++

```diff
@@ -0,0 +1,35 @@
+#define NONIUS_RUNNER
+#include <nonius.h++>
+
+#include <string>
+#include <vector>
+
+NONIUS_PARAM("size", 100u)
+NONIUS_PARAM("string", std::string{"foo"})
+
+NONIUS_BENCHMARK("push_back vector", [](nonius::chronometer meter)
+{
+    auto n = meter.param("size");
+    auto s = meter.param<std::string>("string");
```

Comment:
I had the first option in mind when suggesting this elsewhere, but I must say that I like the second option too, and I can't say I really considered it before for my own uses of this technique. Thanks for the inspiration!